Makefile

```
@@ -54,17 +54,17 @@ $(EMBEDDED_MANIFESTS_TARGET): $(call rwildcard,manifests/,*.yaml *.json)
 	touch $@
 
 build: $(EMBEDDED_MANIFESTS_TARGET)
-	CGO_ENABLED=0 go build -ldflags="-s -w -X main.VERSION=$(VERSION)" -o ./bin/flux ./cmd/flux
+	CGO_ENABLED=1 go build -ldflags="-s -w -X main.VERSION=$(VERSION)" -o ./bin/flux ./cmd/flux
 
 build-dev: $(EMBEDDED_MANIFESTS_TARGET)
-	CGO_ENABLED=0 go build -ldflags="-s -w -X main.VERSION=$(DEV_VERSION)" -o ./bin/flux ./cmd/flux
+	CGO_ENABLED=1 go build -ldflags="-s -w -X main.VERSION=$(DEV_VERSION)" -o ./bin/flux ./cmd/flux
 
 .PHONY: install
 install:
-	CGO_ENABLED=0 go install ./cmd/flux
+	CGO_ENABLED=1 go install ./cmd/flux
```

As we as still flushing this through the repos, add a replace instead:
```suggestion
	replace github.com/pjbgf/sha1cd => github.com/pjbgf/sha1cd v0.2.3
```